deployment/helm/posix-mapper/values.yaml

```diff
@@ -33,17 +38,24 @@ deployment:
     # - containerPort: 5555
     #   protocol: TCP
 
-    # Resources provided to the Skaha service.
+    # -- Resources provided to the Skaha service.
     resources:
       requests:
+        # -- Memory Requested
         memory: "1Gi"
+        # -- CPU Requested
         cpu: "500m"
       limits:
+        # -- Memory Limit
         memory: "1Gi"
+        # -- CPU Limit
         cpu: "500m"
 
+    # -- Minimum UID
     minUID: 10000
+    # -- Minimum GID
     minGID: 900000
```

Comment:
Could you please add to the `minUID` and `minGID` comments that this should be something high so as not to conflict with UIDs and GIDs that may be present in Images.